assets/scss/page.scss

```diff
@@ -0,0 +1,45 @@
+/* Styles - Page Templates */
+
+.page {
+  .page-header {
+
+    width: 100vw;
+    position: relative;
+    left: 50%;
+    right: 50%;
+    margin-left: -50vw;
+    margin-right: -50vw;
+    background: rgba(#000000, 0.1);
+    padding: 1em 0 0 0;
+    margin-bottom: 2em;
+    display: inline-block;
+
+    @include mq($from: tablet) {
+      padding: 2em 0 0 0;
```

Comment:
I think it's fine for now, but we probably want to add in px fallbacks where we can :) it looks like vh support is a bit limited too but I'll take a look when i use it :)